lib/upipe-modules/upipe_rtp_fec.c

```diff
@@ -1,3 +1,23 @@
+/*
+ * Copyright (C) 2015-2017 Open Broadcast Systems Ltd
+ *
+ * Authors: Kieran Kunhya
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
```

Comment:
upipe-modules is MIT, move to upipe-ts or a new library if you want it to be LGPL.